modules/bullet/space_bullet.cpp

```diff
@@ -945,8 +945,8 @@ bool SpaceBullet::test_body_motion(RigidBodyBullet *p_body, const Transform &p_f
 
 	btVector3 motion;
 	G_TO_B(p_motion, motion);
-
-	{ /// phase two - sweep test, from a secure position without margin
+	if (motion.length2() > SIMD_EPSILON * SIMD_EPSILON) {
```

Comment:
Why not use `!motion.fuzzyZero()` directly?

Reply:
I've updated it.